components/ws-daemon/pkg/content/archive.go

```diff
@@ -83,7 +84,9 @@ func BuildTarbal(ctx context.Context, src string, dst string, fullWorkspaceBacku
 	}
 	defer fout.Close()
 	defer func(e *error) {
+		log.Debug("defer called.")
```

Comment:
This is not a good log message because it carries no context. We do not know who's defer was called in which context. Also, defer's are called always even in the event of a panic.

Reply:
yes, it wasn't meant for merging but for debugging an issue.